Makefile.release

```diff
@@ -35,7 +35,7 @@ help:
 
 .PHONY: dist
 dist:                       ## Build distribution
-dist: distdir javascript oc-app distconf sign package
+dist: distdir javascript oc-app distconf sign package docker-image
```

Comment:
Why should that be part of the release makefile?

Reply:
I thought when doing a release having a docker image being spit out is reasonable ....
... this all needs more automation and then the release makefile can die ...